imitation/airl.py

```diff
@@ -105,85 +103,81 @@ def train_gen(self, n_steps=10000):
-        in this array is the observation seen when the generator chooses
-        action `gen_act[i]`.
-      gen_act (array) -- A numpy array with shape
-        `[n_timesteps] + env.action_space.shape`.
-      gen_new_obs (array) -- A numpy array with shape
-        `[n_timesteps] + env.observation_space.shape`. The ith observation
-        in this array is from the transition state after the generator
-        chooses action `gen_act[i]`.
-
-    Return:
-      discriminator_loss (type?) -- The cross-entropy error in the
-        discriminator's clasifications.
+    Args:
+        gen_old_obs (np.ndarray): A numpy array with shape
+            `[n_timesteps] + env.observation_space.shape`. The ith observation
+            in this array is the observation seen when the generator chooses
+            action `gen_act[i]`.
+        gen_act (np.ndarray): A numpy array with shape
+            `[n_timesteps] + env.action_space.shape`.
+        gen_new_obs (np.ndarray): A numpy array with shape
+            `[n_timesteps] + env.observation_space.shape`. The ith observation
+            in this array is from the transition state after the generator
+            chooses action `gen_act[i]`.
+
+    Returns:
+        discriminator_loss (float): The total cross-entropy error in the
+            discriminator's clasifications.
     """
     fd = self._build_disc_feed_dict(**kwargs)
     return np.sum(self._sess.run(self.discrim.disc_loss, feed_dict=fd))
 
   def wrap_env_train_reward(self, env):
-    """
-    Returns the given Env wrapped with a reward function that returns
+    """Returns the given Env wrapped with a reward function that returns
     the AIRL training reward (discriminator confusion).
 
-    The reward network referenced (not copied) into the Env
+    The reward network is referenced (not copied) into the Env
     wrapper, and therefore the rewards are changed by calls to
     AIRLTrainer.train().
 
-    Params:
-    env (str, Env, or VecEnv) -- The Env that we want to wrap. If a
-      string environment name is given or a Env is given, then we first
-      make a VecEnv before continuing.
+    Args:
+        env (str, Env, or VecEnv): The Env that we want to wrap. If a
+            string environment name is given or a Env is given, then we first
+            convert to a VecEnv before continuing.
     """
     env = util.maybe_load_env(env, vectorize=True)
     return _RewardVecEnvWrapper(env, self._policy_train_reward_fn)
 
   def wrap_env_test_reward(self, env):
-    """
-    Returns the given Env wrapped with a reward function that returns
+    """Returns the given Env wrapped with a reward function that returns
     the reward learned by this AIRLTrainer.
 
-    The reward network referenced (not copied) into the Env
-    wrapper, and therefore the rewards are changed by calls to
-    AIRLTrainer.train().
+    References the reward network (not copied) into the `Env`
```

Comment:
Coming from having less context on this project, this isn't super clear: it seems like we're using the reward function attached to self? What's meaningful about it being copied vs not copied?

Reply:
It seems fine to remove the first part and just keep "the wrapped reward is changed when .train() is called".

My documenting motive was to emphasize that the caller should note that the wrapped reward isn't fixed.